### PR TITLE
Support doc critical updates 

### PR DESCRIPTION
**Pull request by itsalyse**
Continues the critical updates of the support docs with 

Reactivate subscription - https://docs.google.com/document/d/1relqCvxZ_v7ncZrvv4HZDVDWCI_sXNABjz1nFvap9dY/edit#heading=h.e4uzflj14h5n

Account invoice history - https://docs.google.com/document/d/1VlQZ6SW7U71Al-is6bl2OMV3nKZgttP8tb55qMd4MNU/edit#heading=h.fb9fl8oumupk

Unsubscribe - https://docs.google.com/document/d/1i17OaM9gCe95U8dRQEjpJMRX_owUnx4ZeWdDEsoHtm8/edit#heading=h.bgm35k979zdt

Understanding your billing - https://docs.google.com/document/d/1_sM1XupifvipAF4LrnTxi0Ir7w7nr-SUaTY1pDfBLdo/edit#heading=h.x5e8nniu6k1p

Change Plan - https://docs.google.com/document/d/1AOb1-6prNnP7x7sohw51N1w3aT_FaNGa33YCNBgk8EE/edit

Related to https://github.com/dnsimple/dnsimple-business/issues/1900

### CODE REVIEW

File: content/articles/account-subscription-balance.md [hunk 1]
```diff
@@ -6,39 +6,43 @@ categories:
...
+- [DNS queries](/articles/dns-query-limits/) 
+- Subscription cost
+- Active zones
+- Extra [seats](/articles/managing-seats/) (on the Teams plan and higher)
+- [Email forwards](/articles/email-forwarding/)
+- Account credits (when applicable)
 
-- **Subscription balance**
-  The current balance of your account. The amount can be:
+**SCREENSHOT**
```
Comment (itsalyse) [inline content/articles/account-subscription-balance.md:29]: https://drive.google.com/drive/u/3/search?q=type:image%20parent:1Y9yg2ZWQDXY8cjvNf6wUiZT8SnPTJ9EP

File: content/articles/account-invoice-history.md [hunk 2]
```diff
@@ -82,12 +74,12 @@ While our system occasionally retries failed invoices automatically, you may wan
 ##### To retry a failed invoice
 
 1. Access your [account's invoice history](#viewing).
-1. Locate failed invoices in your history. They're highlighted in red.
+1. Locate failed invoices in your history, highlighted in red.
 1. Click the retry button on your failed invoice.
 
     ![Retry Button](/files/account-billing-retry-button.png)
```
Comment (itsalyse) [inline content/articles/account-invoice-history.md:80]: we need a new screenshot with the new dashboard here

File: content/articles/changing-plans.md [hunk 3]
```diff
@@ -7,47 +7,71 @@ categories:
...
+## Changing your subscription plan
 
 1.  Log in to DNSimple with your user credentials.
-1.  [Find the account](https://dnsimple.com/user) from the account list. Click on the account to enter the account page.
-1.  Click the <label>Change plan</label> link under your plan section.
+2.  [Find the account](https://dnsimple.com/user) from the account list. Click on the account to enter the account page.
+3.  Click the <label>Change plan</label> link under your plan section.
 
-    ![Change plan](/files/account-billing-change-plan-link.png)
+**New screenshot**
```
Comment (itsalyse) [inline content/articles/changing-plans.md:41]: https://drive.google.com/drive/u/3/folders/1gq_IxXbuszRRYWBWK-lOujG2zxl9oRnw

File: content/articles/changing-plans.md [hunk 4]
```diff
@@ -7,47 +7,71 @@ categories:
...
+3.  Click the <label>Change plan</label> link under your plan section.
 
-    ![Change plan](/files/account-billing-change-plan-link.png)
+**New screenshot** 
 
-1.  Explore the features for each plan. Click the <label>Select</label> button of the plan that best suits your needs.
+4.  Explore the features for each plan. Click the <label>Select</label> button of the plan that best suits your needs.
 
-    ![Identify](/files/change-plan-2.png)
+**new screenshot**
```
Comment (itsalyse) [inline content/articles/changing-plans.md:45]: https://drive.google.com/drive/u/3/folders/1gq_IxXbuszRRYWBWK-lOujG2zxl9oRnw 

New screenshot that doesn't indicate the teams plan is $99/mo

File: content/articles/changing-plans.md [hunk 5]
```diff
@@ -7,47 +7,71 @@ categories:
...
+
+<div class="section-steps" markdown="1">
+To learn if any features you're using are incompatible with your plan change:
+
+1. Log in to your DNSimple account.
+1. Choose the relevant account and navigate to the Account page.
+1. Click **Billing and plans**, then **Change plan**.
+1. You'll be taken to a personalized version of the pricing page. Click **See incompatible features**.
+    
+**Screenshot**
```
Comment (itsalyse) [inline content/articles/changing-plans.md:65]: https://drive.google.com/drive/u/3/folders/1gq_IxXbuszRRYWBWK-lOujG2zxl9oRnw

File: content/articles/reactivate-subscription.md [hunk 6]
```diff
@@ -1,28 +1,52 @@
...
-1.  Locate the accounts section of your dashboard. Click the <label>Choose plan</label> link corresponding to the account you wish to reactivate.
-
-    ![Choose plan link](/files/account-dashboard-reactivation.png)
-
-1.  Choose the plan that best suits your needs.
-1.  Enter your payment information and click <label>Create Subscription</label>.
-</div>
+1.  Locate the accounts section of your dashboard.
+1.  You'll see a card with your account name on it. Click **Select a plan** on this card.
+**new screenshot from accounts section**
```
Comment (itsalyse) [inline content/articles/reactivate-subscription.md:29]: https://drive.google.com/file/d/10t0Sfs4UGaV3eadlfxsZdLrewQSEQdEF/view?usp=share_link

File: content/articles/reactivate-subscription.md [hunk 7]
```diff
@@ -1,28 +1,52 @@
...
+1.  Choose the plan that fits your needs, and click **Get started**.
+1.  Enter your payment information and click **Create Subscription**.
+
+### From the Billings and plans page:
+1. Locate the accounts section of your [dashboard](https://dnsimple.com/dashboard).
+1. Click on the account you want to reactivate, then select **Account** at the top right.
+1. Click the **Billing and plans** tab on the left side of the screen.
+1. You'll see a card with a **No plan selected** alert. Click **select plan** at the bottom of this card.
+**screenshot**
```
Comment (itsalyse) [inline content/articles/reactivate-subscription.md:38]: https://drive.google.com/file/d/1UEkcNBhknt-BnbMnZptbbThhgCHtwyt3/view?usp=share_link

File: content/articles/reactivate-subscription.md [hunk 8]
```diff
@@ -1,28 +1,52 @@
...
+1. You'll see a card with a **No plan selected** alert. Click **select plan** at the bottom of this card.
+**screenshot** 
+1. Choose the plan that fits your needs, and click **Get started**.
+1. Enter your payment information and click **Create Subscription**.
+
+### From the alert banner:
+1. Log in to DNSimple with your user credentials.
+1. Go to your [dashboard](https://dnsimple.com/dashboard).
+1. If your account still has domains stored in it, you'll see an alert banner at the top of the screen. Click **Choose plan**.
+   **banner screenshot**
```
Comment (itsalyse) [inline content/articles/reactivate-subscription.md:46]: I need a screenshot of what this looks like in the banner — I don't have domains in my test account

File: content/articles/account-invoice-history.md [hunk 9]
```diff
@@ -36,15 +36,15 @@ Your invoices may be in certain states depending on different circumstances:
...
 
-When an invoice has been paid for, it'll be in the `collected` state.
+An invoice in the `collected` state indicates it has been paid.
 
 ### `failed` {#invoice-state-failed}
 
-When you make a purchase, like a domain registration or an SSL certificate, and we're unable to collect the payment, we'll notify you via email. The invoice will be marked as `failed`.
+When you make a [purchase](/articles/understanding-invoice/#purchases), like a domain registration or an SSL certificate, and we're unable to collect the payment, we'll notify the account owner via email. The invoice will be marked as `failed`.
 
 While the invoice is in this state, you won't be able to make other purchases. You can [retry the payment](#retrying) yourself, and our systems will also try to collect it every few hours.
```
Comment (itsalyse) [inline content/articles/account-invoice-history.md:49]: "Every few hours" but do we know for how long?

File: content/articles/account-invoice-history.md [hunk 10]
```diff
@@ -55,39 +55,31 @@ Depending on your plan, your subscription will renew every month or every year.
 A `dunned` invoice will automatically be retried once per week for three weeks. You can manually [retry the payment](#retrying) any time. We'll notify you via email on every attempt.
```
Comment (itsalyse) [inline content/articles/account-invoice-history.md:55]: Do we still offer yearly renewals? Have I asked this already?

File: content/articles/account-subscription-balance.md [hunk 11]
```diff
@@ -6,39 +6,43 @@ categories:
 ---
 
 # Account Balance
```
Comment (itsalyse) [inline content/articles/account-subscription-balance.md:8]: Wanted to check that we were all on the same page re: changing this title to **Understanding Your Account Balance and Billing Details**

File: content/articles/account-invoice-history.md (hunk 10 above)
Comment (stephanoodle) [reply content/articles/account-invoice-history.md:55]: Only for people who are still on a legacy plan (Personal, Professional). 

The Solo and Teams plan are only monthly.

File: content/articles/account-invoice-history.md (hunk 9 above)
Comment (stephanoodle) [reply content/articles/account-invoice-history.md:49]: I do not have a specific number. It seems variable. Maybe we should put something like " and our systems will also try to collect it automatically again."

File: content/articles/account-subscription-balance.md (hunk 11 above)
Comment (stephanoodle) [reply content/articles/account-subscription-balance.md:8]: LGTM. I like the title change.

**Review by stephanoodle — APPROVED** [review]

**Review by stephanoodle — APPROVED** [review]

File: content/articles/account-subscription-balance.md (hunk 1 above)
Comment (dperrymorrow) [reply content/articles/account-subscription-balance.md]: have added the screenshot and pushed up

**Comment by dperrymorrow** [comment]
@itsalyse i added all the drive images referenced to the site, and sized and compressed them. I am not sure I know how to get screenshots of the failed invoices, or the resubscribe banner warning either...

**Comment by itsalyse** [comment]
@dperrymorrow let's not worry about those right now then, they're not crucial to wrapping up the updates.

File: content/articles/reactivate-subscription.md (hunk 6 above)
Comment (itsalyse) [reply content/articles/reactivate-subscription.md:29]: Screenshot still needs to be added

File: content/articles/reactivate-subscription.md (hunk 7 above)
Comment (itsalyse) [reply content/articles/reactivate-subscription.md:38]: screenshot still needs to be added

**Review by stephanoodle — APPROVED** [review]

**Review by Pemacaa — APPROVED** [review]

File: content/articles/reactivate-subscription.md (hunk 6 above)
Comment (itsalyse) [reply content/articles/reactivate-subscription.md:29]: @dperrymorrow could you add this screenshot?

File: content/articles/reactivate-subscription.md (hunk 7 above)
Comment (itsalyse) [reply content/articles/reactivate-subscription.md:38]: @dperrymorrow could you add this screenshot?

**Comment by dperrymorrow** [comment]
> Could you add the two I was able to get for the reactivate your account article?

these have been added and pushed to this branch

**Review by dperrymorrow — APPROVED** [review]

**Pull request merged**